src/Resolvers/VisibleDataFieldsResolver.php

```diff
@@ -214,6 +214,7 @@ protected function resolveIncludedFields(
                 continue;
             }
 
+            $includedPartial = clone $includedPartial;
```

Comment:
This is what I found to fix the issue, the problem is that it's looping over the elements of the collection but all the elements of the collection get the same instance of `includedPartial` which is mutated with `includedPartial->next()` and so `$includedPartial->getNested()` returns null on the second item